lib/apiservers/engine/backends/container.go

```diff
@@ -153,16 +152,17 @@ func (c *Container) ContainerCreate(config types.ContainerCreateConfig) (types.C
 
 	// bail early if container name already exists
 	if exists := cache.ContainerCache().GetContainer(config.Name); exists != nil {
-		return types.ContainerCreateResponse{},
-			derr.NewErrorWithStatusCode(fmt.Errorf("Conflict. The name \"%s\" is already in use by container %s. You have to remove (or rename) that container to be able to re use that name.", config.Name, exists.ContainerID),
-				http.StatusConflict)
+		err := fmt.Errorf("Conflict. The name \"%s\" is already in use by container %s. You have to remove (or rename) that container to be able to re use that name.", config.Name, exists.ContainerID)
+		log.Errorf("%s", err.Error())
+		return types.ContainerCreateResponse{}, derr.NewErrorWithStatusCode(err, http.StatusConflict)
 	}
 
 	// get the image from the cache
 	image, err := cache.ImageCache().GetImage(config.Config.Image)
 	if err != nil {
 		// if no image found then error thrown and a pull
 		// will be initiated by the docker client
+		log.Errorf("ContainerCreate: image not found %s", config.Config.Image)
```

Comment:
No idea. I just wanted to log the error since it was completely silent previously.